2018/TeamCode/src/main/java/com/terrabats/ftc/teamcode/TerraBot.java

```diff
@@ -8,9 +8,12 @@
 import com.qualcomm.robotcore.hardware.DcMotorSimple;
 import com.qualcomm.robotcore.hardware.HardwareMap;
 import com.qualcomm.robotcore.hardware.Servo;
+import com.qualcomm.robotcore.util.ElapsedTime;
 
+import static java.lang.Thread.sleep;
 
-public class TerraBot {
+
+public class Terrabot {
```

Comment:
The class name should match the file name:
```suggestion
public class TerraBot {
```